code/modules/telesci/telepad.dm

```diff
@@ -51,27 +51,41 @@
 
 	return ..()
 
-
 //CARGO TELEPAD//
 /obj/machinery/telepad_cargo
 	name = "cargo telepad"
-	desc = "A telepad used by the Rapid Crate Sender."
+	desc = "A telepad used by the cargo transporter."
 	icon = 'icons/obj/telescience.dmi'
-	icon_state = "pad-idle"
-	anchored = 1
-	use_power = 1
+	icon_state = "pad-idle-o"
+	anchored = TRUE
+	use_power = TRUE
 	idle_power_usage = 20
 	active_power_usage = 500
+
 	var/stage = 0
+	var/active = FALSE
+
+/obj/machinery/telepad_cargo/New()

```

Comment:
/tg/ added initialize for station objects, mind taking a look if that might apply here?